### PR TITLE
feat: Create sidebar

### PR DESCRIPTION
**Pull request by quyxuan**
- Before
image
- After
image

### CODE REVIEW

**Review by DinhManhAVG — APPROVED** [review]
Good mr

**Review by DinhManhAVG — COMMENTED** [review]
PR will be merged

**Review by taiphanvan2k3 — DISMISSED** [review]

File: src/app/components/dashboard/body/body.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (taiphanvan2k3) [inline src/app/components/dashboard/body/body.component.spec.ts:2]: remove empty line

File: src/app/components/dashboard/body/body.component.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-body',
+  templateUrl: './body.component.html',
+  styleUrls: ['./body.component.css'],
+})
+export class BodyComponent {
```
Comment (taiphanvan2k3) [inline src/app/components/dashboard/body/body.component.ts:8]:
```suggestion

export class BodyComponent {
```

File: src/app/components/dashboard/body/chat-page/chat-page.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (taiphanvan2k3) [inline src/app/components/dashboard/body/chat-page/chat-page.component.spec.ts:2]: remove empty line

File: src/app/components/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
+  screenWidth: number;
+  collapsed: boolean;
+}
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
-  styleUrls: ['./dashboard.component.css']
+  styleUrls: ['./dashboard.component.css'],
 })
 export class DashboardComponent {
```
Comment (taiphanvan2k3) [inline src/app/components/dashboard/dashboard.component.ts:12]:
```suggestion

export class DashboardComponent {
```

File: src/app/components/dashboard/side-bar/side-bar.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (taiphanvan2k3) [inline src/app/components/dashboard/side-bar/side-bar.component.spec.ts:2]: remove empty line

File: src/app/components/dashboard/side-bar/side-bar.component.ts [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+  screenWidth: number;
+  collapsed: boolean;
+}
+
+@Component({
+  selector: 'app-side-bar',
+  templateUrl: './side-bar.component.html',
+  styleUrls: ['./side-bar.component.css'],
+})
+export class SideBarComponent implements OnInit {
```
Comment (taiphanvan2k3) [inline src/app/components/dashboard/side-bar/side-bar.component.ts:29]:
```suggestion

export class SideBarComponent implements OnInit {
```

File: src/app/components/dashboard/body/body.component.spec.ts (hunk 1 above)
Comment (quyxuan) [reply src/app/components/dashboard/body/body.component.spec.ts:2]: k can thiệp vô file .spec

File: src/app/components/dashboard/body/body.component.ts (hunk 2 above)
Comment (quyxuan) [reply src/app/components/dashboard/body/body.component.ts:8]: new component thì nó như vậy sẵn rồi

**Review by taiphanvan2k3 — APPROVED** [review]

**Pull request merged**